lang/pl/index.md

```diff
@@ -64,80 +64,146 @@ i „OPCJONALNY” („OPTIONAL”) w tym dokumencie należy interpretować jak
 w [RFC 2119](http://tools.ietf.org/html/rfc2119).
 
 1. Oprogramowanie używające wersjonowania semantycznego MUSI określać swoje
-publiczne API. API to może być zadeklarowane w samym kodzie lub może istnieć
-w samej dokumentacji. Jakkolwiek jest zdefiniowane, powinno być precyzyjne
-i wyczerpujące.
+   publiczne API. API to może być zadeklarowane w samym kodzie lub może istnieć
+   w samej dokumentacji. Jakkolwiek jest zdefiniowane, powinno być precyzyjne
+   i wyczerpujące.
 
 2. Standardowy numer wersji MUSI przyjąć formę X.Y.Z, gdzie X, Y i Z są
-nieujemnymi liczbami całkowitymi i NIE MOGĄ zawierać wiodących zer. X jest
-wersją major, Y wersją minor, a Z wersją patch. Każdy składnik MUSI rosnąć
-numerycznie. Przykładowo: 1.9.0 → 1.10.0 → 1.11.0.
+   nieujemnymi liczbami całkowitymi i NIE MOGĄ zawierać wiodących zer. X jest
+   wersją major, Y wersją minor, a Z wersją patch. Każdy składnik MUSI rosnąć
+   numerycznie. Przykładowo: 1.9.0 → 1.10.0 → 1.11.0.
 
-3. Po wydaniu wersjonowanego pakietu, zawartość tej wersji NIE MOŻE być
-modyfikowana. Jakiekolwiek zmiany MUSZĄ być wydane jako nowa wersja.
+3. Po wydaniu wersjonowanego pakietu zawartość tej wersji NIE MOŻE być
+   modyfikowana. Jakiekolwiek zmiany MUSZĄ być wydane jako nowa wersja.
 
 4. Wersja major zero (0.y.z) jest przeznaczona dla początkowej fazy rozwoju.
-Wszystko może ulec zmianie w dowolnym momencie. Publiczne API nie powinno być
-traktowane jako stabilne.
+   Wszystko może ulec zmianie w dowolnym momencie. Publiczne API nie powinno być
+   traktowane jako stabilne.
 
 5. Wersja 1.0.0 określa publiczne API. Sposób, w jaki numer wersji jest
-zwiększany po tym wydaniu, zależy od tego publicznego API i jak się ono zmienia.
+   zwiększany po tym wydaniu, zależy od tego publicznego API i jak się ono zmienia.
 
 6. Wersja patch Z (x.y.Z | x > 0) MUSI zostać zwiększona, jeśli wprowadza się
-tylko kompatybilne wstecz naprawy błędów. Naprawa błędu definiowana jest jako
-zmiana wewnętrzna, która usuwa nieprawidłowe działanie.
+   tylko kompatybilne wstecz naprawy błędów. Naprawa błędu definiowana jest jako
+   zmiana wewnętrzna, która usuwa nieprawidłowe działanie.
 
 7. Wersja minor Y (x.Y.z | x > 0) MUSI zostać zwiększona, jeśli nowa,
-kompatybilna wstecz funkcjonalność zostaje wprowadzona do publicznego API. MUSI
-zostać zwiększona, jeśli jakakolwiek funkcjonalność publicznego API zostaje
-zdezaprobowana. MOŻE zostać zwiększona, jeśli wprowadzone zostają nowe znaczące
-funkcjonalności lub ulepszenia w obrębie prywatnego kodu. MOŻE ona zawierać
-zmiany na poziomie patch. Numer wersji patch MUSI być ustawiony na 0, gdy wersja
-minor jest zwiększana.
+   kompatybilna wstecz funkcjonalność zostaje wprowadzona do publicznego API. MUSI
+   zostać zwiększona, jeśli jakakolwiek funkcjonalność publicznego API zostaje
+   wycofane. MOŻE zostać zwiększona, jeśli wprowadzone zostają nowe znaczące
```

Comment:
"Wycofana" means withdrawn, which isn't the same as deprecated. Deprecate means "expressing a disapproval".

"[funkcjonalność] zostaje oznaczona jako wycofywana" would be a synonim here, but I'm not quite sure if it's better then current "zostaje zdezaprobowana".

Reply:
A word `zdezaprobowana` does not exist. 
In theory, the best matching word is `usunięta`, but that will be problematic with `How should I handle deprecating functionality?` section.